client/scss/utilities/_functions.scss

```diff
@@ -3,3 +3,27 @@
 
   @return map-get($colors, $name);
 }
+
+// Converts pixel values to rems 
+@function px-to-rem($pixels, $base: $base-font-size) {
+  @if (unitless($pixels)) {
+    $pixels: $pixels * 1px;
+  }
+
+  @if (unitless($base)) {
+    $base: $base * 1px;
+  }
+
+  @return $pixels / $base * 1rem;
+}
+
+// Converts Photoshop tracking value to CSS em equivalent.
+// https://scotch.io/tutorials/converting-photoshop-letter-spacing-to-css
+@function tracking-to-em($tracking-value) {
```

Comment:
I'd be tempted to not start on the wrong foot - nont sure photoshop should be creeping into the code base.